src/core/device.c

```
@@ -32,6 +32,27 @@ static const UnitActiveState state_translation_table[_DEVICE_STATE_MAX] = {
 
 static int device_dispatch_io(sd_device_monitor *monitor, sd_device *dev, void *userdata);
 
+static int device_by_path(Manager *m, const char *path, Unit **ret) {
+        _cleanup_free_ char *e = NULL;
+        Unit *u;
+        int r;
+
+        assert(m);
+        assert(path);
+
+        r = unit_name_from_path(path, ".device", &e);
+        if (r < 0)
+                return r;
```

Can we keep the debug log here that was removed from the two call sites?

---

I dropped the log in response to https://github.com/systemd/systemd/pull/24522#discussion_r962896850.
I will consider to add log message in caller side.

---

I noticed all caller ignore the error.